### ISSUE TITLE
Improve Windows embedding DPI support

### ISSUE THREAD

**Issue by stuartmorgan** [flutter/flutter#38599]
@clarkezone Could you summarize the work that needs to be done here? I know this was an area you identified as needing follow-up in the initial patch.

**Comment by clarkezone** [comment]
will do

**Comment by ye4241** [comment]
Same issue on macOS if two monitors with different scale.
The window will be blurry if move from high dpi monitor to low dpi monitor.

**Comment by stuartmorgan** [comment]
@ye4241 You're looking for #35095; this is completely unrelated.

**Comment by clarkezone** [comment]
I think this can be closed now that the DPI code is correctly conditionalized for downlevel
